config/routes.rb

```diff
@@ -46,7 +46,13 @@
     resources :bt_wifi_vouchers, only: %i[index], path: '/bt-wifi-vouchers' do
       get 'download', to: 'bt_wifi_vouchers#download', on: :collection
     end
-    resources :extra_mobile_data_requests, only: %i[index new create], path: '/extra-mobile-data-requests'
+    namespace :mobile, path: '/mobile' do
+      get '/', to: 'extra_data_requests#index', as: :extra_data_requests
+      get '/type', to: 'extra_data_requests#new', as: :extra_data_requests_type
+      post '/type', to: 'extra_data_requests#create', as: :extra_data_requests_type_select
```

Comment:
I can see what you mean, we are submitting the chosen upload type so think it could stay as a `post`? I would normally try and stick with the 'standard' CRUD methods but I'm happy to change to something more meaningful.

Reply:
I agree with sticking to the standard CRUD methods when you're doing CRUD actions - but this isn't creating anything. The only reason it's a form at all is to present the user with a radio button, the flow would work just as well with two links. I'm _not_ suggesting that it actually _should_ be two links, by the way (the radio button UI works well and is govuk-standard), my point is that because it _could_ work that way, that's a sure sign that it fits better to GET requests.

Reply:
Done